docs/build/apps/stellar-wallets-kit.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 🚧 Stellar Wallets Kit
+sidebar_position: 80
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+This document introduces the user to the [Stellar Wallets Kit]('https://stellarwalletskit.dev/'). The Stellar Wallets Kit is a library that simplifies handling multiple Stellar wallets through a straightforward API. With this kit, you can interact with various wallets without worrying about individual configurations. Developers can focus on UI/UX design while relying on this library for wallet connections.
+
+## Compattible Wallets:
+
+The following wallets are compatible with the Stellar Wallets Kit:
+
+- xBull Wallet (both PWA and extension versions)
+- Albedo
+- Freighter
+- Rabet (extension version)
+- WalletConnect
+- Lobstr
+- Hana
+
+## Installation
+
+To install the library, use npm:
+
+<CodeExample>
+  ```bash npm install @creit.tech/stellar-wallets-kit ```
+</CodeExample>
+
+## Usage
+
+### Creating an Instance
+
+1. First, create an instance of the `StellarWalletsKit` class. You should only create one instance to avoid unexpected behavior. Here’s an example:
+
+<CodeExample >
+```javascript
+import {
+  StellarWalletsKit,
+  WalletNetwork,
+  allowAllModules,
+  XBULL_ID,
+} from '@creit.tech/stellar-wallets-kit';
+
+const kit = new StellarWalletsKit({ network: WalletNetwork.TESTNET, selectedWalletId: XBULL_ID, modules: allowAllModules(), });
+
+````
+</CodeExample>
+The `allowAllModules()` function imports modules where no additional configuration is needed (e.g., WalletConnect). If you want to specify only certain wallets to support, pass an array of modules to the constructor like this :
+<CodeExample >
+```javascript
+import {
+  FreighterModule,
+  StellarWalletsKit,
+  WalletNetwork,
+  XBULL_ID,
+  xBullModule
+} from '@creit.tech/stellar-wallets-kit';
+
+const kit: StellarWalletsKit = new StellarWalletsKit({
+  network: WalletNetwork.TESTNET,
+  selectedWalletId: XBULL_ID,
+  modules: [
+    new xBullModule(),
+    new FreighterModule(),
```

Comment:
For this example of wallet set up, could we include all wallet modules for completeness. We should especially show the set up for WalletConnect module for completeness.